stylesheet.css

```diff
@@ -0,0 +1,214 @@
+body {
+    margin: 10px;
+    padding: 0;
+    background-color: whitesmoke;
+    font-family: Roboto, Arial, sans-serif;
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: white;
+}
+
+.homeBar {
+    width: 100%;
+    overflow: auto;
+
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+.logo img{
+    width: 100px;
+    height:50%;
+}
+
+.searchLink {
+    display: flex;
+    width: 65%;
+    align-items: center;
+    padding-left: 10%;
+    flex-wrap: wrap;
+}
+
+.searchBar {
+    width: 75%;
+    height: 25px;
+}
+
+.icons {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    height: 25px;
+    padding-right: 10px;
+}
+ 
+.fab {
+    color: red;
+    font-size: 35px;
+}
+
+.fas {
+    padding: 0 5px 0 5px;
+    font-size: 20px;
+    height: 25px;
+}
+
+.container {
+    display: flex;
+    display: -webkit-flex;
+    justify-content: space-around;
+    margin: 0 2px 0 2px;
+    ;
+}
+
+.right-container {
+    float: right;
+    align-items: center;
+    width: 35%;
+    overflow: 0;
+}
+
+.left-container {
+    width: 70%;
+    float: left;
+    margin-right: 20px;
+}
+
+.userData {
+    background-color: white;
+    width: 100%;
+}
+
+#subscribe {
+    background-color: red;
+    color: white;
+}
+
+.channel {
+    display: flex;
+    flex-direction: none;
+}
+
+.channel1.channel2 {
+    float: left;
+}
+
+.channel3 {
+    float: right;
+    padding: 5% 1px 0 50%;
+}
+
+.small {
+    border-color: black;
+    border-style: solid;
+    border: 2px;
+}
+
+.share {
+    display: flex;
+}
+.share-btn{
+    background-color:white;
+    border-style: none;
+}
+.share-right {
+    display: flex;
+    padding-left: 55%;
+}
+
+.share-left {
+    display: flex;
+}
```

Comment:
- [ ] Remove all the unnecessary repeated CSS code ([Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) Tip: Create one only class like this: .d-flex{ display: flex } and use that same class in your index.html when you need it.